Runners/TelegramPullRunner/Handler.cs

```diff
@@ -45,25 +41,15 @@ public async Task HandleUpdateAsync(ITelegramBotClient botClient, Update update,
             }
         }
 
-        private Task BotOnMessageReceived(ITelegramBotClient botClient, Message message)
+        private static Task BotOnMessageReceived(ITelegramBotClient botClient, Message message)
```

Comment:
Its preferable to not use the 'static' keyword, its may create a global state, this is not desirable